docs/reference/export-metrics.mdx

```diff
@@ -163,3 +163,42 @@ You can stop the metrics collection at any time by deleting the gadget instance:
         ```
     </TabItem>
 </Tabs>
+
+## Other Exporters
+
+If you have more complex requirements for metric exporting, you can setup multiple exporters using the
+config file. In this section we're exploring other exporters.
+
+### OTLP-gRPC
+
+In order to setup an exporter using [OTLP](https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/otlp/) over
+gRPC, you can add the following section to your config file:
+
+```yaml
```

Comment:
I test it with the following commands.

Server:

```bash
$ sudo cat /root/.ig/config.yaml
operator:
  otel-metrics:
    exporters:
      myexporter:
        exporter: otlp-grpc
        endpoint: "localhost:4320"
        insecure: true
        temporality: cumulative
        interval: 5s
$ go run -exec "sudo" ./cmd/ig/... daemon
INFO[0000] starting Inspektor Gadget daemon at "unix:///var/run/ig/ig.socket"
WARN[0000] no TLS configuration provided, communication between daemon and CLI will not be encrypted
```

Client:
```bash
$ go run -exec "sudo" ./cmd/gadgetctl/... run ghcr.io/inspektor-gadget/gadget/profile_blockio:latest                         --annotate=blockio:metrics.collect=true --otel-metrics-name blockio:blockio-metrics --otel-metrics-exporter myexporter
```

I use the [otel-col](https://opentelemetry.io/docs/collector/) to check the pushed data:

```bash
$ cat config.yaml
receivers:
  otlp:
    protocols:
      grpc:
        endpoint: localhost:4320
exporters:
  debug:
    verbosity: detailed
service:
  pipelines:
    metrics:
      receivers: [otlp]
      exporters: [debug]
$ otelcol --config config.yaml
[...]
2024-11-04T15:40:05.740+0100    info    Metrics {"kind": "exporter", "data_type": "metrics", "name": "debug", "resource metrics": 1, "metrics": 1, "data points": 1}
2024-11-04T15:40:05.740+0100    info    ResourceMetrics #0
Resource SchemaURL: https://opentelemetry.io/schemas/1.26.0
Resource attributes:
     -> service.name: Str(unknown_service:gadgetctl)
     -> telemetry.sdk.language: Str(go)
     -> telemetry.sdk.name: Str(opentelemetry)
     -> telemetry.sdk.version: Str(1.31.0)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope blockio-metrics
Metric #0
Descriptor:
     -> Name: latency
     -> Description:
     -> Unit:
     -> DataType: Histogram
     -> AggregationTemporality: Cumulative
HistogramDataPoints #0
StartTimestamp: 2024-11-04 14:39:52.842051559 +0000 UTC
Timestamp: 2024-11-04 14:40:05.740396733 +0000 UTC
Count: 1689
Sum: 4188680.000000
Min: 8.000000
Max: 8192.000000
ExplicitBounds #0: 1.000000
ExplicitBounds #1: 2.000000
ExplicitBounds #2: 4.000000
ExplicitBounds #3: 8.000000
ExplicitBounds #4: 16.000000
ExplicitBounds #5: 32.000000
ExplicitBounds #6: 64.000000
ExplicitBounds #7: 128.000000
ExplicitBounds #8: 256.000000
ExplicitBounds #9: 512.000000
ExplicitBounds #10: 1024.000000
ExplicitBounds #11: 2048.000000
ExplicitBounds #12: 4096.000000
ExplicitBounds #13: 8192.000000
ExplicitBounds #14: 16384.000000
ExplicitBounds #15: 32768.000000
ExplicitBounds #16: 65536.000000
ExplicitBounds #17: 131072.000000
ExplicitBounds #18: 262144.000000
ExplicitBounds #19: 524288.000000
ExplicitBounds #20: 1048576.000000
ExplicitBounds #21: 2097152.000000
ExplicitBounds #22: 4194304.000000
ExplicitBounds #23: 8388608.000000
ExplicitBounds #24: 16777216.000000
ExplicitBounds #25: 33554432.000000
ExplicitBounds #26: 67108864.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 13
Buckets #4, Count: 182
Buckets #5, Count: 232
Buckets #6, Count: 71
Buckets #7, Count: 123
Buckets #8, Count: 110
Buckets #9, Count: 240
Buckets #10, Count: 109
Buckets #11, Count: 112
Buckets #12, Count: 99
Buckets #13, Count: 398
Buckets #14, Count: 0
Buckets #15, Count: 0
Buckets #16, Count: 0
Buckets #17, Count: 0
Buckets #18, Count: 0
Buckets #19, Count: 0
Buckets #20, Count: 0
Buckets #21, Count: 0
Buckets #22, Count: 0
Buckets #23, Count: 0
Buckets #24, Count: 0
Buckets #25, Count: 0
Buckets #26, Count: 0
Buckets #27, Count: 0
[...]
```

Notes:

- The CLI doesn't print the histogram to the terminal. If I don't set `--otel-metrics-exporter`, it is printed. Why?
- Unit are empty and not `µs`.